tests/cupy_tests/linalg_tests/test_product.py

```diff
@@ -256,14 +258,16 @@ def test_transposed_tensordot(self, xp, dtype):
         b = testing.shaped_arange((4, 3, 2), xp, dtype).transpose(2, 0, 1)
         return xp.tensordot(a, b)
 
-    @testing.for_all_dtypes()
+    @testing.for_all_dtypes(excludes=(numpy.uint8, numpy.int8, numpy.uint16,
```

Comment:
Thank you for replying. I'm afraid I don't know what you exactly mean. What do you mean by "you need to fix the test"? I intended to fix the test by this patch, IOW, this already is trying to "fix the test". Do you mean that I need to fix the test "in a different way"?